identity/src/test/java/com/android/identity/UtilTest.java

```diff
@@ -403,7 +410,7 @@ public void cborCalendarForeign() throws CborException {
                 .add("2019-07-08T11:51:42.25Z")
                 .build());
         data = baos.toByteArray();
-        assertEquals("tag 0 '2019-07-08T11:51:42.250Z'",
+        assertEquals("tag 0 '2019-07-08T11:51:42.025Z'",
```

Comment:
Why have these strings changed? Looking at line 410, it looks like this should still be .250Z instead of .025Z?

Reply:
Because this is non-standard, so parsing it is ambiguous.

OpenJDK seems to interpret the `.` as a delimiter, and everything after is milliseconds. So `.1234` means "1.234 seconds". Likewise `.25` is "25 milliseconds".

Android seems to interpret the `.` as a decimal point. So `.1234` means "123.4 milliseconds".

So these tests are going to get different results for different runtimes :/ Both actually seem reasonable to me.

If we're not ok with this, then I see a few options:
1. We fail if the input isn't well-formed. This is my preferred option, because it means that we avoid any possible risks due to misinterpretation of malformed timestamps.
2. Roll our own parsing. No way, haha. I'd much rather go with #1
3. Take a dependency on some other third party parser with well-defined behavior so we're not at the mercy of the runtime.

Reply:
We just don't parse millis anymore -- we don't need them. 18013-5 even says fractional seconds are not allowed.